Automattic/wp-calypso
# Migrate the stacked comparison views to the new mailbox list

#### Proposed Changes

* Conditionally converts components `<GoogleWorkspaceCard/>` and `<ProfessionalEmailCard/>` to use the `<NewMailBoxList/>` component, based on the `unify-mailbox-forms` feature flag

#### Testing Instructions



* Ensure you have a domain without an email subscription
* Click on the [ **+Add** ] from the domain management view i.e. [ **Upgrades** ] > [ **Domains** ] > [ + **Add**  ] . Alternatively navigate to `/email/<:domain>/purchase/<:siteSlug>`
* Append `?flags=unify-mailbox-forms` to the end of the URL to activate the `unify-mailbox-forms` feature flag.
* Confirm that the form and validation works as usual.
* Confirm that you now see an option to add multiple mailboxes to the new subscription.
* Confirm that you can checkout successfully.

Note: Unique mailboxes on the same form are not validated yet.

##### The Professional Email view
<img width="700" alt="Screenshot 2022-06-21 at 8 00 13 AM" src="https://user-images.githubusercontent.com/277661/174736597-a01d6475-a5b7-433d-9356-fe125da9f4cf.png">




Related to #64424 and #64746
